bay_trimesh/anaconda-project.yml

```diff
@@ -0,0 +1,53 @@
+name: bay_trimesh
+description: Visualizing water depth into the Chesapeake and Delaware Bays
+maintainers:
+- philippjfr
+
+channels:
+- pyviz
+
+dependencies: &deps
+- colorcet ==2.0.1
+- datashader ==0.7.0
+- geoviews ==1.6.2
+- holoviews ==1.12.2
+- notebook ==5.7.8
+- pandas ==0.24.2
+
+packages: *deps
+
+platforms:
+- linux-64
+- osx-64
+- win-64
+
+commands:
+  notebook:
+    notebook: bay_trimesh.ipynb
+    env_spec: pyviz-topics-bay-trimesh
+  test:
+    unix: pytest --nbsmoke-run -k *.ipynb --ignore envs
+    windows: pytest --nbsmoke-run -k *.ipynb --ignore envs
+    env_spec: test
+  lint:
+    unix: pytest --nbsmoke-lint -k *.ipynb --ignore envs
+    windows: pytest --nbsmoke-lint -k *.ipynb --ignore envs
+    env_spec: test
+
+variables: {}
+downloads:
+  DATAFILE:
+    url: https://s3.amazonaws.com/datashader-data/Chesapeake_and_Delaware_Bays.zip
+    filename: data
+    unzip: true
+
+env_specs:
+  pyviz-topics-bay-trimesh:
+    channels: []
+    packages: []
```

Comment:
@jsignell I have stumbled upon this file as a good example for how to use anaconda-project.yml for topics. This might have already been resolved as a question (I've not been following any discussion here) but do you need `channels: []` and `packages: []` here? Haven't they been declared at the top level?

Reply:
I will do a test.

Reply:
works :)

Reply:
You say it works, but those lines still seem to be here?  Same for the pyviz channel spec below?